services/hierarchy/src/main/kotlin/PackageService.kt

```diff
@@ -52,20 +51,23 @@ class PackageService(private val db: Database) {
         }
     }
 
-    suspend fun countForOrtRunId(ortRunId: Long): Long = db.dbQuery {
+    /** Count packages found in provided ORT runs. */
+    suspend fun countForOrtRunIds(vararg ortRunIds: Long): Long = db.dbQuery {
         PackagesTable.joinAnalyzerTables()
             .select(PackagesTable.id)
-            .where { AnalyzerJobsTable.ortRunId eq ortRunId }
+            .where { AnalyzerJobsTable.ortRunId inList ortRunIds.toList() }
```

Comment:
Prefer `asList()` over `toList()` here, as the former does not make a copy, but just provides another "view" on the same data. Similar in other places.